src/containers/BooksList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
```

Comment:
thanks for the hooks approach eye-opener, i appreciate.....learnt from your project 😊